src/components/inputs/InlineField.tsx

```diff
@@ -42,9 +42,14 @@ export default function InlineField({
   const handleContentSizeChange = useCallback(
     ({ nativeEvent }) => {
       if (inputProps?.multiline) {
-        const contentHeight = nativeEvent.contentSize.height;
+        const contentHeight = ios
+          ? nativeEvent.contentSize.height
+          : nativeEvent.contentSize.height - 32.8571421305;
```

Comment:
hm this is probably different fo different android devices? wdyt @jxom

Reply:
I'm not entirely sure. Where does the 32.85 value come from? @benisgold

Reply:
ok so with ios, the contentSize.height starts at ~19 and increases by 19 with every line. on my android, it starts at like ~52 and increases by some constant (i forget) with every line. I subtract 32.85 because then the contentSize.height of just the first line is equivalent to the height of every other line, instead of being disproportionately large.